azure-pipelines.yaml

```diff
@@ -27,3 +27,23 @@ stages:
           solution: VectorRumble.Desktop/VectorRumble.Desktop.csproj
           configuration: Release
           msbuildArguments: /restore /v:diag
+    - job: BuildUWP
+      displayName: UWP
+      pool:
+        vmImage: vs2017-win2016
```

Comment:
windows-latest throws

> VectorRumble.UWP\VectorRumble.UWP.csproj(0,0): Error WMC1006: Cannot resolve Assembly or Windows Metadata file 'Windows.Foundation.UniversalApiContract.winmd'
> C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Microsoft\WindowsXaml\v16.0\8.2\Microsoft.Windows.UI.Xaml.Common.targets(266,5): Error WMC9999: Type universe cannot resolve assembly: Windows.Foundation.UniversalApiContract, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null, ContentType=WindowsRuntime.
So this is the only image I could use to get it green.